app/scripts/require_config.js

```diff
@@ -25,7 +25,9 @@ require.config({
     crosstab: 'vendor/crosstab',
     uuid: '../bower_components/node-uuid/uuid',
     'jquery-simulate': '../bower_components/jquery-simulate/jquery.simulate',
-    raven: '../bower_components/raven-js/dist/raven'
+    raven: '../bower_components/raven-js/dist/raven',
+    bloomfilter: '../bower_components/fxa-password-strength-checker/src/bloomfilter',
```

Comment:
Delcaring bloomfilter here breaks localizing context, bloomfilter is not used by the content-server, but rather, it's used by the fxa-password-strength-checker and should be moved there. The fxa-password-strength-checker may need a build step.